pages/resources.tsx

```diff
@@ -71,11 +110,24 @@ function ResourceSection(props: ResourceSectionProps) {
         />
         <span>{title}</span>
       </Heading>
-      <SimpleGrid mt={8} columns={[1, 2]} spacing={8}>
-        {resources.map((item, index) => (
-          <ResourceCard key={index} data={item} />
-        ))}
-      </SimpleGrid>
+      <FormControl id={filterInputId} mt='8' mb='8'>
+        <FormLabel>{t('resources.searchFilter.label')}</FormLabel>
+        <Input
+          onChange={formik.handleChange}
+          placeholder={t('resources.searchFilter.placeholder')}
+          value={formik.values[filterInputId]}
+        />
+        <FormHelperText>{t('resources.searchFilter.helperText')}</FormHelperText>
+      </FormControl>
+      <ResponsiveMasonry columnsCountBreakPoints={{ 350: 1, 580: 2 }}>
+        <Masonry gutter='15px'>
```

Comment:
Let's use values from our design tokens. Even though they're in pixels, make them `16px` so we can have consistent spacing.

Reply:
Good catch!  I've committed the update, so that now we are using `16px`.